modules/swagger-codegen/src/main/resources/Java/libraries/feign/pom.mustache

```diff
@@ -161,7 +161,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <swagger-core-version>1.5.8</swagger-core-version>
```

Comment:
@zhenjun115  Yes, it's. That's why I want you to help upgrade it to the latest version.

In other words, I want you to upgrade all dependencies to the latest stable version.